plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/LibraryClasspathContainerInitializer.java

```diff
@@ -102,6 +101,7 @@ private static boolean jarPathsAreValid(LibraryClasspathContainer container) {
     return true;
   }
 
+  // todo may be unused
```

Comment:
This should have had `@Override`. I vaguely remember @briandealwis added this method.

Reply:
Guess I turned off that warning. Fixed.